lib/OpenQA/Utils.pm

```diff
@@ -899,4 +900,6 @@ sub find_video_files { path(shift)->list_tree->grep(VIDEO_FILE_NAME_REGEX) }
 # workaround https://github.com/mojolicious/mojo/issues/1629
 sub fix_top_level_help { @ARGV = () if ($ARGV[0] // '') =~ qr/^(-h|(--)?help)$/ }
 
+sub is_url { return shift =~ /http(s)?\:\/\//; }
```

Comment:
```suggestion
sub is_http_url { shift =~ /http(s)?\:\/\// }
```

Reply:
maybe you got it the wrong way around, see https://github.com/os-autoinst/openQA/pull/3696#discussion_r564587717 . I think the method name was correct, the implementation was wrong and should cover more than just http(s)